spec/features/user_show_spec.rb

```diff
@@ -3,12 +3,12 @@
 RSpec.describe 'users/show.html.erb', type: :feature do
   describe 'show page' do
     let(:user) { User.create(name: 'Tom', photo: 'https://picsum.photos/id/64/200', bio: 'Teacher from Mexico', post_counter: 0) }
-    let!(:post1) { Post.create(user:, title: 'Post 1', text: 'This is my first post', author_id: user.id) }
-    let!(:post2) { Post.create(user:, title: 'Post 2', text: 'This is my second post', author_id: user.id) }
-    let!(:post3) { Post.create(user:, title: 'Post 3', text: 'This is my third post', author_id: user.id) }
+    let!(:post1) { Post.create(user:, title: 'Post 1', text: 'This is my first post', author_id: 1) }
+    let!(:post2) { Post.create(user:, title: 'Post 2', text: 'This is my second post', author_id: 2) }
+    let!(:post3) { Post.create(user:, title: 'Post 3', text: 'This is my third post', author_id: 3) }
```

Comment:
- [ ] Great job so far working on the recommended changes by the previous reviewer. However, as this test indicates to show the first 3 posts of a user, we should create at least 4 posts and test our method. As mentioned by the previous reviewer, please do so 🙏